### ISSUE TITLE
Multi and complex parameters

### ISSUE THREAD

**Issue by Ladylain** [mrjeanp/Arcane-SEO#12]
Hi, 

i'm using your plugin for octobercms, and i need to generate a dynamic sitemap. but i'm using complex parameters on my cms pages like this : 
`/blog/:page?/:slug*`

your plugin generates this kind of urls : 
`https://dev.alpine-lodges.fr/blog?/espace-killy*`

I think the `Helper::replaceUrlPlaceholders()` function could be improved. what do you think ? 

best,

**Comment by mrjeanp** [comment]
yes, I thought about it when i was making the plugin... seemed very difficult for me
is a good idea

**Comment by tomaszstrojny** [comment]
@Ladylain @mrjeanp Is there any continuation regarding the issue? If not, we're about to create a PR that handles this. I am thinking about something like a map of models and parameters, maybe using a repeater field?

Something like this:
```php
'slug' => [
    'Acme\Example\Class' => 'slug'
],
'page' => [
    'Acme\Example\Page' => 'slug'
],
```

@progowski
